plugins/TraceUpdates/TraceUpdatesAbstractNodePresenter.js

```diff
@@ -131,7 +136,7 @@ class TraceUpdatesAbstractNodePresenter {
 
     this.drawImpl(this._pool);
 
-    if (this._pool.size > 0) {
+    if (this._pool.size > 0 && this._clearTimer != null) {
```

Comment:
I believe this introduced a bug. The `setTimeout` call inside should've executed regardless of whether `this._clearTimer` exists.

Reply:
fixed in https://github.com/facebook/react-devtools/pull/1005